source/index.rst

```diff
@@ -1,92 +1,86 @@
-
-Contents
-========
+The Sponge project currently offers two Implementations, **SpongeForge** and **SpongeVanilla**.
+Both make use of our **SpongeAPI**.
+
+If you want to dive into Sponge real quick, have a look at our :doc:`server/getting-started/index` section.
+You'll learn how to setup your server there. If you want to help us developing Sponge or you're planning to create your
+own plugins, then :doc:`plugin/index` and :doc:`contributing/index` are the right places to start. If you want to know
+more about Sponge, then give :doc:`about/index` a try.
+
+If you just want to download SpongeForge or SpongeVanilla, then head to :doc:`downloads`.
+
+.. note::
+
+    Sponge is still in development. Development builds are currently available for :doc:`download <downloads>`, but an official
+    release will not be available until later in 2015.
+
++--------------------+--+-----------------------+
+| Table of Contents                             |
++====================+==+=======================+
+|.. toctree::        |  |.. toctree::           |
+|    :maxdepth: 2    |  |    :maxdepth: 2       |
+|    :titlesonly:    |  |    :titlesonly:       |
+|                    |  |                       |
+|    server/index    |  |    contributing/index |
+|                    |  |                       |
+|.. toctree::        |  |.. toctree::           |
+|    :maxdepth: 2    |  |    :maxdepth: 2       |
+|    :titlesonly:    |  |    :titlesonly:       |
+|                    |  |                       |
+|    preparing/index |  |    about/index        |
+|                    |  |                       |
+|.. toctree::        |  |.. toctree::           |
+|    :maxdepth: 2    |  |    :maxdepth: 2       |
+|    :titlesonly:    |  |    :titlesonly:       |
+|                    |  |                       |
+|    plugin/index    |  |    downloads          |
+|                    |  |                       |
++--------------------+--+-----------------------+
+
+
+
+The Sections in Detail
+======================
 
 Creating a Server
 ~~~~~~~~~~~~~~~~~
 
-This section is for users who have a Sponge server or are interested in creating one.
-
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
-
-    server/index
+This section is for users who have a Sponge server or are interested in creating one. It tells you how to install Java,
+get Sponge set up correctly and will even show you how what Sponge provides to help you out, while managing your server.
 
 Preparing for Development
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section is for:
-
-* Developers who wish to create Sponge plugins or contribute code to the Sponge project
-* Users who wish to help with writing and editing articles in SpongeDocs
-
-Please read this section before `Creating Plugins`_ and `Contributing to Sponge`_.
-
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
+This section is covers the basic setup (and how to do it) of your workspace wether you want to develop for Sponge your
```

Comment:
is covers; wether; "develop for Sponge your own plugins" -> "develop plugins for Sponge"